### PR TITLE
check privilege translations via mypy

### PR DESCRIPTION
**Pull request by petrjasek**
it needs stubs for flask_babel and actually validating imports

### CODE REVIEW

File: apps/publish/__init__.py [hunk 1]
```diff
@@ -68,15 +71,33 @@ def init_app(app):
     service = UnpublishService(endpoint_name, backend=get_backend())
```
Comment (GyanP) [inline apps/publish/__init__.py:71]: this should be Any as well

File: apps/saved_searches/__init__.py [hunk 2]
```diff
@@ -47,24 +50,30 @@ def init_app(app):
     service = SavedSearchItemsService(endpoint_name, backend=superdesk.get_backend())
```
Comment (GyanP) [inline apps/saved_searches/__init__.py:50]: same here

**Review by GyanP — APPROVED** [review]

File: apps/publish/__init__.py (hunk 1 above)
Comment (petrjasek) [reply apps/publish/__init__.py:71]: what do you mean?

Comment (petrjasek) [reply apps/publish/__init__.py:71]: it must be set only when service is first declared in the function

**Pull request merged**